packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -294,12 +329,17 @@ const gateway = mesh.addVirtualGateway('gateway', {
         interval: cdk.Duration.seconds(10),
       },
   })],
+  backendDefaultsClientPolicy: appmesh.ClientPolicy.fileTrust({
+    certificateChain: 'path-to-certificate',
+  }),
 });
 ```
 
 The listeners field can be omitted which will default to an HTTP Listener on port 8080.
 A gateway route can be added using the `gateway.addGatewayRoute()` method.
 
+The `backendDefaults` property are added to the node while creating the virtual gateway. These are virtual gateway's service backends client policy defaults.
```

Comment:
Old name used here (it should be `backendDefaultsClientPolicy`, not `backendDefaults`).